plugins/pulp_rpm/plugins/migrations/0031_regenerate_repo_unit_counts.py

```diff
@@ -45,6 +45,6 @@ def migrate(*args, **kwargs):
     """
     db = connection.get_database()
     repos_collection = db['repos']
-    for repo in repos_collection.find():
-        repo_id = repo['repo_id']
+    repo_ids = [x['repo_id'] for x in repos_collection.find()]
```

Comment:
This is actually pymongo and not mongoengine. Please use a raw mongodb query to limit the fields returned.